examples/dashboard/dashboard.py

```diff
@@ -135,6 +191,8 @@ class AppState(object):
     def __init__(self, config_file, outofcore, app_port):
 
         self.load_config_file(config_file)
+        self.plot_height = 560
+        self.plot_width = 810
```

Comment:
Can the plot just be set to fill the browser window, or is that hard from bokeh Python?